src/UserContext.ts

```diff
@@ -0,0 +1,10 @@
+import { createContext } from 'react';
+import type { User } from '@pdc/sdk';
+
+const emptyUser = {
+	id: 0,
+	authenticationId: '',
+	createdAt: new Date(0),
+};
```

Comment:
One small change I'd probably make is export this empty user so we can reuse it in App.tsx instead of redeclaring it.